src/lava/proc/scif/process.py

```diff
@@ -23,7 +23,8 @@ def __init__(
             shape: ty.Tuple[int, ...],
             step_size: ty.Optional[int] = 1,
             theta: ty.Optional[int] = 4,
-            neg_tau_ref: ty.Optional[int] = -5) -> None:
+            neg_tau_ref: ty.Optional[int] = -5,
```

Comment:
Why such arbitrary values? If it's truly arbitrary then they typical value would be 0.

Reply:
The arbitrary negative value is to check refractory period. But yes, that belongs to a unittest. Default initialization can be -1 or 0.